modules/rostests/apitests/user32/PrivateExtractIcons.c

```diff
@@ -39,6 +39,7 @@ START_TEST(PrivateExtractIcons)
         ahIcon = (HICON)UlongToHandle(0xdeadbeef);
         aIconId = 0xdeadbeef;
         cIcons = PrivateExtractIconsW(IconTests[i].FilePath, 0, 16, 16, &ahIcon, &aIconId, 1, 0);
+        ok(cIcons != 0xFFFFFFFF, "PrivateExtractIconsW(%u): File couldn't be found (or invalid PE image) - %S\n", i, IconTests[i].FilePath);
```

Comment:
This line is redundant?
Since cIcons is already checked against `IconTests[i].cIcons`